db/upgrade_migrations/20160723012349_upgrade_v0_6_to_v0_7.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class UpgradeV06ToV07 < ActiveRecord::Migration
+  def up
+    Thredded::MessageboardGroup.transaction do
+      Thredded::MessageboardGroup.where(
+        name: Thredded::MessageboardGroup.group(:name).having('count(id) > 1').select(:name)
+      ).group_by(&:name).each_value do |messageboard_groups|
+        messageboard_groups.each_with_index do |messageboard_group, i|
+          messageboard_group.update!(name: "#{messageboard_group.name}-#{i + 2}")
+        end
+      end
+    end
```

Comment:
If the suffix to the group name doesn't end up looking like

* group-name
* group-name-1
* group-name-2

Then could we assign a random int suffix to the duplicate name columns with pure sql?

random int between 1 and 10

```
postgres=# SELECT floor(random()*(10-1+1))+1;
 ?column?
----------
     8
(1 row)
```

append to column value

```
update the_table
  set the_column = concat(the_column, 'this will be appended')
where
```

had to look these up but this should be possible with pure sql and might be a little easier to follow then the pure ruby code above.

1. http://www.techonthenet.com/postgresql/functions/random.php
2 http://stackoverflow.com/questions/30162486/append-text-to-column-data-in-postgresql-if-columns-initial-value-is-null